depends/packages/qt.mk

```diff
@@ -1,53 +1,114 @@
 PACKAGE=qt
-$(package)_version=5.2.1
-$(package)_download_path=http://download.qt-project.org/official_releases/qt/5.2/$($(package)_version)/single
-$(package)_file_name=$(package)-everywhere-opensource-src-$($(package)_version).tar.gz
-$(package)_sha256_hash=84e924181d4ad6db00239d87250cc89868484a14841f77fb85ab1f1dbdcd7da1
+$(package)_version=5.4.2
+$(package)_download_path=http://download.qt.io/official_releases/qt/5.4/$($(package)_version)/submodules
```

Comment:
If you don't trust verification by SHA256 baked into the download code, then you really shouldn't trust HTTPS ;)